doc/conf.py

```diff
@@ -1,22 +1,18 @@
-# Configuration file for the Sphinx documentation builder.
-#
-# This file only contains a selection of the most common options. For a full
-# list see the documentation:
-# https://www.sphinx-doc.org/en/master/usage/configuration.html
-
-# -- Path setup --------------------------------------------------------------
-
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#
-import os
-import sys
-#sys.path.insert(0, os.path.abspath('.'))
-sys.path.insert(0, os.path.abspath('..'))
-#sys.path.insert(0, os.path.abspath('../pyIPXACT'))
-#sys.path.insert(0, os.path.abspath('_extensions'))
-#sys.path.insert(0, os.path.abspath('_themes/sphinx_rtd_theme'))
+# documentation root, use abspath to make it absolute, like shown here.
+
+from sys import path as sys_path
+from os.path import abspath
+from pathlib import Path
+from json import loads
+
+ROOT = Path(__file__).resolve().parent
+
+sys_path.insert(0, abspath('..'))
+#sys_path.insert(0, abspath('../pyIPXACT'))
+#sys_path.insert(0, abspath('_extensions'))
+#sys_path.insert(0, abspath('_themes/sphinx_rtd_theme'))
 
 
 # -- Project information -----------------------------------------------------
```

Comment:
Where is pyTooling to extract `__***__` variables?

Reply:
I tried to do the minimal changes to have a site on github pages and to merge for testing the pypi name change.  There are further enhacements to be done. Apart from using pyTooling, the codecov and codacy shields are disabled, doc license is missing, etc.